### PR TITLE
Update to proposal process

### PR DESCRIPTION
**Pull request by mhdawson**
Update to staging process based on discussion in 
CPC meeting

### CODE REVIEW

**Comment by mhdawson** [comment]
Node: from discussion in the CPC meeting we will also need to
restructure the existing proposal directory

**Comment by joesepi** [comment]
I guess when you do the work in the Github UI, it doesn't show as a renamed file with updates to that file. Oh well. Regardless, I think we tweaked enough of the language to just look at is as a fresh file as shown in the pull request.

File: PROPOSAL_PROCESS.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+They should have a rough description of the program idea or policy, what resources would be needed to execute it, what committee or role
+is responsible for overseeing it, ideas for metrics, execution, and so forth. They have clear language about which group or groups
+the program/policy will serve. The purpose of this stage is to capture the spirit of an idea the community may want to explore
+further, and to begin thinking about implementation ideas or process questions that need to be answered. 
+
+The PR should include a new directory in the proposals directory titled for the proposal with a README.md with the outline 
+for the proposal.
+
+An incubating proposal can be updated through ss many PRs as you’d like, essentially providing snapshots
```
Comment (ljharb) [inline PROPOSAL_PROCESS.md:23]:
```suggestion
An incubating proposal can be updated through as many PRs as you’d like, essentially providing snapshots
```

**Review by ljharb — APPROVED** [review]

**Review by joesepi — APPROVED** [review]
Thanks. We can restructure the files/folders once we have more reviewers.

**Review by dmethvin — APPROVED** [review]

File: PROPOSAL_PROCESS.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Ready for Approval
+
+When ready a new PR is created to move an incubating proposal to the ready for approval by the CPC and if necessary the board.
+Proposals in this stage are written up in the style of the formal policy or program language. They include examples of the
+program idea or policy in action, with clear steps and guidelines. They include the steps that will need to be taken to
+initially implement the idea or policy. 
+
+The purpose of this stage is to refine the policy, make sure all
+stakeholders (including the board) have had a chance to review and request changes, and determine whether further iteration is needed.
```
Comment (eemeli) [inline PROPOSAL_PROCESS.md:33]:
```suggestion
stakeholders (potentially including the board) have had a chance to review and request changes, and determine whether further iteration is needed. 
```

**Review by eemeli — CHANGES_REQUESTED** [review]
The change request is just for the possibly-misplaced line. Overall I definitely approve of this.

As a bit of a dogfooding exercise, should this change follow its own suggested process?

**Review by eemeli — APPROVED** [review]

**Pull request merged**